app/controllers/auctions_controller.rb

```diff
@@ -1,53 +1,51 @@
 class AuctionsController < ApplicationController
-  before_action :is_admin, except: [:index, :show, :bid]
-
+  before_action :is_admin, except: [:index, :show, :bid, :new]
   before_action :authenticate_user!, only: [:bid]
-
+  expose :auctions, -> { Auction.all }
```

Comment:
expose :auctions should replace the whole index method (yes you have to expose also `q`)